### DISCUSSION TITLE
Bluelink - South Korea

### DISCUSSION THREAD

**Discussion by gavind87** [Hyundai-Kia-Connect/kia_uvo#482] [Ideas]
Hi
I live in South Korea and would be keen to have this running off my Home Assistant. Is there any way that can be done with this ?

**Comment by cdnninja** [comment]
Currently we support 4 APIs. If you sniff the traffic I'm happy to give it a shot in adding it.   How to sniff the traffic is in our wiki.   If it had a website that can control the car that makes it easier since you can use the edge method.
